utils/build-windows-toolchain.bat

```diff
@@ -824,193 +129,7 @@ git clone --quiet --no-tags --depth 1 --branch v1.2.11 https://github.com/madler
-  -D Foundation_DIR=%BuildRoot%\4\cmake\modules ^
-
-  -D ENABLE_TESTING=YES ^
-  -D XCTEST_PATH_TO_LIBDISPATCH_BUILD=%BuildRoot%\3 ^
-  -D XCTEST_PATH_TO_LIBDISPATCH_SOURCE=%SourceRoot%\swift-corelibs-libdispatch ^
-  -D XCTEST_PATH_TO_FOUNDATION_BUILD=%BuildRoot%\4 ^
-
-  -G Ninja ^
-  -S %SourceRoot%\swift-corelibs-xctest || (exit /b)
-cmake --build %BuildRoot%\5 || (exit /b)
-
-:: Test XCTest
-cmake --build %BuildRoot%\5 --target check-xctest || (exit /b)
-
-goto :eof
-endlocal
-
-:PackageToolchain
-setlocal enableextensions enabledelayedexpansion
-
-:: Package Online Installer
-msbuild %SourceRoot%\swift-installer-scripts\platforms\Windows\bundle\installer.wixproj ^
-  -m ^
-  -restore ^
-  -p:BundleFlavor=online ^
-  -p:BaseReleaseDownloadUrl=https://download.swift.org/development/windows ^
-  -p:Configuration=Release ^
-  -p:BaseOutputPath=%PackageRoot%\online\ ^
-  -p:VCRedistInstaller="%VCToolsRedistDir%\vc_redist.%VSCMD_ARG_TGT_ARCH%.exe" ^
-  -p:VSVersion=%VSCMD_VER% ^
-  -p:DEVTOOLS_ROOT=%BuildRoot%\Library\Developer\Toolchains\unknown-Asserts-development.xctoolchain ^
-  -p:TOOLCHAIN_ROOT=%BuildRoot%\Library\Developer\Toolchains\unknown-Asserts-development.xctoolchain ^
-  -p:PLATFORM_ROOT=%PlatformRoot%\ ^
-  -p:SDK_ROOT=%SDKInstallRoot%\
-:: TODO(compnerd) actually perform the code-signing
-:: signtool sign /f Apple_CodeSign.pfx /p Apple_CodeSign_Password /tr http://timestamp.digicert.com /fd sha256 %PackageRoot%\online\installer.exe
-
-:: Package Offline Installer
-msbuild %SourceRoot%\swift-installer-scripts\platforms\Windows\bundle\installer.wixproj ^
-  -m ^
-  -restore ^
-  -p:BundleFlavor=offline ^
-  -p:Configuration=Release ^
-  -p:BaseOutputPath=%PackageRoot%\offline\ ^
-  -p:VCRedistInstaller="%VCToolsRedistDir%\vc_redist.%VSCMD_ARG_TGT_ARCH%.exe" ^
-  -p:VSVersion=%VSCMD_VER% ^
-  -p:DEVTOOLS_ROOT=%BuildRoot%\Library\Developer\Toolchains\unknown-Asserts-development.xctoolchain ^
-  -p:TOOLCHAIN_ROOT=%BuildRoot%\Library\Developer\Toolchains\unknown-Asserts-development.xctoolchain ^
-  -p:PLATFORM_ROOT=%PlatformRoot%\ ^
-  -p:SDK_ROOT=%SDKInstallRoot%\
-:: TODO(compnerd) actually perform the code-signing
-:: signtool sign /f Apple_CodeSign.pfx /p Apple_CodeSign_Password /tr http://timestamp.digicert.com /fd sha256 %PackageRoot%\installer\installer.exe
-
-:: Stage Artifacts
-md %BuildRoot%\artifacts\online
-md %BuildRoot%\artifacts\offline
-
-move %PackageRoot%\offline\Release\amd64\installer.exe %BuildRoot%\artifacts\offline || (exit /b)
-move %PackageRoot%\online\Release\amd64\*.cab %BuildRoot%\artifacts\online\ || (exit /b)
-move %PackageRoot%\online\Release\amd64\*.exe %BuildRoot%\artifacts\online\ || (exit /b)
-move %PackageRoot%\online\Release\amd64\*.msi %BuildRoot%\artifacts\online\ || (exit /b)
-
-:: Workaround for lack of control over Jenkins ...
-copy %BuildRoot%\artifacts\offline\installer.exe %BuildRoot%\artifacts\
+git clone --quiet --no-tags --depth 1 --branch curl-7_77_0 https://github.com/curl/curl
```

Comment:
This needs to be part of update-checkout, we should not clone repos outside of update-checkout.

Reply:
Also, this version is wrong it should 8.4.x